src/main/java/com/squareup/javapoet/AnnotationSpec.java

```diff
@@ -221,7 +222,9 @@ public Builder addMember(String name, CodeBlock codeBlock) {
      * depending on the given {@code value} object. Falls back to {@code "$L"} literal format if
      * the class of the given {@code value} object is not supported.
      */
-    private Builder addValue(String memberName, Object value) {
+    public Builder addMemberForValue(String memberName, Object value) {
```

Comment:
I’m not sure. This change is tricky because it brings us to having two different ways to do everything.